src/components/Avatar/Avatar.vue

```diff
@@ -136,6 +136,13 @@ export default {
 			type: Boolean,
 			default: false
 		},
+		/**
+		 * Disable the menu
```

Comment:
Indentation looks odd :wink:

Reply:
Does it? it's ok here :thinking: 
![Capture d’écran_2019-08-02_12-22-37](https://user-images.githubusercontent.com/14975046/62363777-3b4d1880-b520-11e9-8652-3a9494d6ddf8.png)

Reply:
Ah sorry for the noise, refined github was rendering that wrong